.github/workflows/exercise-7-pipeline.yml

```diff
@@ -14,3 +14,20 @@ jobs:
     - uses: actions/checkout@v2
     - run: docker build --tag todo-app:test --target test .
     - run: docker run todo-app:test
+
+  deploy:
+    name: Push to Docker Hub and trigger app service deploy
+    runs-on: ubuntu-latest
+    needs: build
+    if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/exercise-9' }}
```

Comment:
(Don't forget to update this to point to your main branch when you're ready to merge)